test_data/grape_peach_synteny/config.json

```diff
@@ -52,6 +52,20 @@
         "assemblyNames": ["peach", "grape"]
       },
       "assemblyNames": ["peach", "grape"]
+    },
+    {
+      "type": "SyntenyTrack",
```

Comment:
might be good to add the GFF track for this so you can see the features line up

Reply:
i tried adding e.g.

```
{
      "type": "FeatureTrack",
      "trackId": "grape_genes",
      "name": "Vvinifera_457_v2.1.gene.sorted.gff",
      "adapter": {
        "type": "Gff3TabixAdapter",
        "gffGzLocation": {
          "uri": "https://s3.amazonaws.com/jbrowse.org/genomes/grape/Vvinifera_457_v2.1.gene.sorted.gff.gz"
        },
        "index": {
          "location": {
            "uri": "https://s3.amazonaws.com/jbrowse.org/genomes/grape/Vvinifera_457_v2.1.gene.sorted.gff.gz.tbi"
          }
        }
      },
      "assemblyNames": ["grape"]
    },
    {
      "type": "FeatureTrack",
      "trackId": "peach_genes",
      "name": "Ppersica_298_v2.1.gene.sorted.gff3",
      "adapter": {
        "type": "Gff3TabixAdapter",
        "gffGzLocation": {
          "uri": "https://s3.amazonaws.com/jbrowse.org/genomes/peach/Ppersica_298_v2.1.gene.sorted.gff3.gz"
        },
        "index": {
          "location": {
            "uri": "https://s3.amazonaws.com/jbrowse.org/genomes/peach/Ppersica_298_v2.1.gene.sorted.gff3.gz.tbi"
          }
        }
      },
      "assemblyNames": ["peach"]
    },
```

but nothing was lining up

Reply:
I've added these tracks